wizard/WizardDaemonSettings.qml

```diff
@@ -92,6 +92,35 @@ ColumnLayout {
                     blockchainFolder.focus = true;
                 }
             }
+            onTextUpdated: {
+                existingDbWarning.visible = daemonManager.checkLmdbExists(persistentSettings.blockchainDataDir)
+            }
+        }
+
+        RowLayout {
+            id: pruningOptionRow
+            MoneroComponents.CheckBox {
+                id: pruneBlockchainCheckBox
+                checked: persistentSettings.pruneBlockchain
+                onClicked: {
+                    if (daemonManager.checkLmdbExists(persistentSettings.blockchainDataDir)) {
+                        persistentSettings.pruneBlockchain = false
+                        this.checked = false
+                        return
+                    }
+                    persistentSettings.pruneBlockchain =  !persistentSettings.pruneBlockchain
+                    this.checked = persistentSettings.pruneBlockchain
+                }
+                text: qsTr("Prune blockchain") + translationManager.emptyString
+            }
+
+            Text {
+                id: existingDbWarning
+                text: "A blockchain database already exists here. Select a new location to start a pruned node"
+                visible: daemonManager.checkLmdbExists(persistentSettings.blockchainDataDir)
```

Comment:
daemonManager can be undefined / null. Same change at other places daemonManager is used.

```suggestion
                visible: daemonManager ? daemonManager.checkLmdbExists(persistentSettings.blockchainDataDir) : false
```